app/Http/Kernel.php

```diff
@@ -108,6 +109,7 @@ class Kernel extends HttpKernel
         'can' => Authorize::class,
         'bindings' => SubstituteBindings::class,
         'recaptcha' => VerifyReCaptcha::class,
+        'maintenance' => MaintenanceMiddleware::class,
```

Comment:
I'm wondering if we should re-label this as `NodeMaintenanceMode` (`node.maintenance`) so its not confused with the Laravel specific maintenance mode.